HW03/Igor_Hrybinyk/lib/mentor.rb

```diff
@@ -0,0 +1,35 @@
+class Mentor
+  include StudentObserver
+  attr_reader :mentor_name
+  attr_accessor :students, :mentor_notify_count
```

Comment:
```suggestion
  attr_accessor :students, :notifications
```
It seems to me better to use such a variable name, because notifications will store some kind of message